src/python/bcc/table.py

```diff
@@ -412,9 +412,37 @@ def __init__(self, *args, **kwargs):
         raise Exception("Unsupported")
 
 class StackTrace(TableBase):
+    MAX_DEPTH = 127
```

Comment:
@4ast it makes sense to me to leave this unless the kernel value is ever likely to change. How likely is that?

The problem with getting this dynamically will be that the kernel headers may update but the libc-headers, where the userspace value of this will be most readily available, is very slow to update. I tend to come across (and am one myself) lagging userspace header files, which is why bcc includes a bunch of compat header files and hardcoded things.